values.yaml

```diff
@@ -1,397 +1,165 @@
-  enable: true
-
-# serviceTolerations -- Tolerations are applied to all Coder managed services.
-# Each element is a toleration object. To set user environment tolerations see
-# environments.tolerations. See values.yaml for an example.
-#
-# e.g.
-# serviceTolerations:
-# - key: "my key"
-#   operator: "Exists"
-#   effect: "NoExecute"
-#   tolerationSeconds: 6000
-serviceTolerations: []
+# Contains fields related to the Postgres backend. If providing your own
+# instance, a minimum version of Postgres 11 is required with the contrib
+# package installed.
+postgres:
+  # postgres.host -- Host of the external PostgreSQL instance.
+  host: ""
+  # postgres.port -- Port of the external PostgreSQL instance.
+  port: ""
+  # postgres.user -- User of the external PostgreSQL instance.
+  user: ""
+  # postgres.database -- Name of the database that Coder will use.
+  # You must create this database first.
+  database: ""
+  # postgres.passwordSecret -- Name of an existing secret in the
+  # current namespace with the password of the PostgreSQL instance.
+  # The password must be contained in the secret field `password`.
+  # This should be set to an empty string if the database does not
+  # require a password to connect.
+  passwordSecret: ""
+  # postgres.sslMode -- Provides variable levels of protection for
+  # the PostgreSQL connection. For acceptable values, see:
+  #  https://www.postgresql.org/docs/9.1/libpq-ssl.html
+  sslMode: "require"
 
-# Services contains global configuration that applies to all Coder managed
-# services.
+  # postgres.default -- Configure a built-in PostgreSQL deployment.
+  default:
+    # postgres.default.enable -- Deploys a PostgreSQL instance. We recommend
+    # using an external PostgreSQL instance in production.
+    # If true, all other values are ignored.
+    enable: true
+    # postgres.default.image -- Injected by Coder during release.
+    image: ""
+    # postgres.default.storageClassName -- Set the storageClass to store
+    # the database.
+    storageClassName: ""
+    # postgres.default.resources -- Kubernetes resource specification for the PostgreSQL pod.
+    # To unset a value, set it to "". To unset all values, set resources to nil.
+    resources:
+      requests:
+        cpu: "250m"
+        memory: "1Gi"
+        # postgres.default.resources.requests.storage -- Specifies the size of the volume claim
+        # for persisting the database.
+        storage: "10Gi"
+      limits:
+        cpu: "250m"
+        memory: "1Gi"
+
+# services -- Kubernetes Service configuration that applies to Coder services. 
 services:
-  # services.nodeSelector -- Node selectors that will be applied to all Coder
-  # managed services.
-  # See: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
+  # services.annotations -- A KV mapping of annotations. See:
+  # https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+  annotations: {}
```

Comment:
since `annotations` is now under a general `services` value, what is the syntax for specifying a specific service?
I expect a user would need to define the service first, then include the service-specific annotation after the fact, such as:

```yaml
loadbalancer.cloud.google.com/load-balancer-type: "Internal"
```

or 

```yaml
nginx.ingress.kubernetes.io/ssl-redirect: "true"
```

but would like to confirm.

Reply:
Since there isn't an ingress anymore, I didn't feel like it was necessary.

Before we used `deploymentAnnotations`, which applied to all Coder services too.

Reply:
You did find a bug though... I wasn't respecting the `services.annotations` value in the Coder service. So tyty

Reply:
Err I guess I'm confused... how did we do this before to annotate a service? If we didn't that's all good... just curious

Reply:
I'm not exactly sure, I never set annotations outside of the ingress object/service (hence my question above about the syntax). 

Also, it looks like `serviceAccount.annotations` was [used to set annotations for all Coder services](https://github.com/cdr/enterprise-helm/blob/be124a4e5b2839f31834c7ee1fb4b59699b164cc/values.yaml#L383). `deploymentAnnotations` doesn't have any comments about it.

Reply:
Going to merge for now... we'll find out much more bugs I'm sure in testing.